.github/workflows/code-ql-analysis.yml

```diff
@@ -16,5 +16,5 @@ jobs:
 
     - name: Run CodeQL
       run: |
-        docker run --rm -v $PWD:/app composer sh -c \
+        docker run --rm -v $PWD:/app composer:2.6 sh -c \
```

Comment:
Required to make codeql job pass, otherwise it runs in php 8.4 and fails with phpstan version